web/src/composables/useLogs.ts

```
@@ -4129,6 +4161,171 @@ const useLogs = () => {
+          left: left,
+          right: right,
+        };
+      }
+
+      case "column_ref":
+        return {
+          type: "column_ref",
+          table: node.table || null,
+          column:
+            node.column && node.column.expr
+              ? node.column.expr.value
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)